cluster/gce/gci/configure-helper.sh

```diff
@@ -1410,6 +1410,13 @@ function start-kube-apiserver {
 
   if [[ -n "${ENCRYPTION_PROVIDER_CONFIG:-}" ]]; then
     local encryption_provider_config_path="/etc/srv/kubernetes/encryption-provider-config.yml"
+
+    if [[ -n "${GOOGLE_CLOUD_KMS_CONFIG_FILE_NAME:-}" ]]; then
+        if [[ -n "${GOOGLE_CLOUD_KMS_CONFIG:-}" ]]; then
```

Comment:
Inside `[[ ]]` you can use `( )` and `&&` which is what you want IIUC

Reply:
Used `if [[ -n cond1 && -n cond2 ]]` as is used in other parts of the script. Going to amend commit.